docs/en/pid-eaa-data-model.rst

```diff
@@ -19,14 +19,16 @@ The User attributes carried in the Italian PID are:
 
 The italian PID is extended according to the `OpenID Identity Assurance Profile [OIDC.IDA] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html>`_, that enables the binding of the PID to a national trust framework, giving all the evidence of the identity proofing procedures underlying the PID issuing in both remote and proximity flows.
 
-The PID data format and the mechanism through which it is issued into the Wallet Instance and presented to a RP will be detailed in the next sections. 
+The (Q)EAAs are issued by the (Q)EAA Issuers to a Wallet Instance and MUST be provided in SD-JWT-VC or mDOC CBOR data format. They MAY contain any (qualified) attributes, entitlement or any elettronic attestations. 
```

Comment:
```suggestion
The (Q)EAAs are issued by the (Q)EAA Issuers to a Wallet Instance in SD-JWT-VC or mDOC CBOR data format.
```

Reply:
I kept the normative language here. @peppelinux let me know if it is fine with you.